wf/entrypoint.py

```diff
@@ -0,0 +1,430 @@
+import csv
+import os
+import shutil
+import subprocess
+import sys
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Annotated, List, Optional
+
+import requests
+from flytekit.core.annotation import FlyteAnnotation
+from latch.executions import rename_current_execution, report_nextflow_used_storage
+from latch.ldata.path import LPath
+from latch.resources.tasks import custom_task, nextflow_runtime_task
+from latch.types.directory import LatchDir, LatchOutputDir
+from latch.types.file import LatchFile
+from latch_cli.nextflow.utils import _get_execution_name
+from latch_cli.nextflow.workflow import get_flag
+from latch_cli.services.register.utils import import_module_by_path
+from latch_cli.utils import urljoins
+
+from wf.enums import DADATaxonomy, Kraken2RefTaxonomy, QiimeRefTaxonomy, SidleRefTaxonomy, SintaxRefTaxonomy
+
+meta = Path("latch_metadata") / "__init__.py"
+import_module_by_path(meta)
+
+sys.stdout.reconfigure(line_buffering=True)
+
+
+@dataclass(frozen=True)
+class SampleSheet:
+    sampleid: str
+    forwardreads: LatchFile
+    reversereads: Optional[LatchFile]
+    run: Optional[str]
```

Comment:
should be snake cased

ex. sampleid -> sample_id 
forwardreads -> forward_reads

Reply:
Would have liked to but I am trying to maintain convention with the original nfcore as closely as possible because people might get confused if we use different name

<img width="1012" alt="image" src="https://github.com/user-attachments/assets/ff1dcce0-86e4-409c-8f88-3c1ae870a6c7">